### PR TITLE
Updated DOB Date Format in Patient details page

### LINKED ISSUE #6830: Change DOB(date) Format in Patient Details Page
**Describe the bug**
On the Patient Details Page the date of birth is shown in the format of YYYY-MM-DD it would be better if we show it in DD-MM-YYYY
https://care.ohc.network/facility/eeb23cb7-1524-479a-825e-660311386b47/patient/9ed7c221-a3ac-4bb3-8fcb-f0e0756cb49e

**To Reproduce**
1. Go to patient's tab
2. Click on any patient
3. See the error

**Expected behavior**
show dob in the format of DD-MM-YYYY

**Screenshots**
image

### PR DESCRIPTION
**Pull request by AdityyaX**
## Proposed Changes

- Fixes #6830 
- Format of DOB in patient section changed to DD/MM/YYYY Format

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Screenshot
Screenshot 2023-12-30 at 10 18 52 AM

made the requested changes @rithviknishad

### CODE REVIEW

File: src/Components/Patient/PatientHome.tsx [hunk 1]
```diff
@@ -85,6 +85,20 @@ export const PatientHome = (props: any) => {
...
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const year = date.getFullYear();
+
+    const formattedDay = day < 10 ? `0${day}` : day;
+    const formattedMonth = month < 10 ? `0${month}` : month;
+
+    return `${formattedDay}/${formattedMonth}/${year}`;
+  }
+
```
Comment (rithviknishad) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
```

File: src/Components/Patient/PatientHome.tsx [hunk 2]
```diff
@@ -572,7 +586,7 @@ export const PatientHome = (props: any) => {
                     Date of Birth
                   </div>
                   <div className="mt-1 text-sm font-medium leading-5">
-                    {patientData?.date_of_birth}
+                    {formatDateOfBirth(patientData?.date_of_birth)}
```
Comment (rithviknishad) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                    {formatDate(patientData?.date_of_birth)}
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]
We already have a reusable utility function called: `formatDate` that does this for you.

**Comment by AdityyaX** [comment]
@rithviknishad 
Okay i have made the suggested changes

**Review by rithviknishad — COMMENTED** [review]
Are you sure you've imported the required method?

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**